studio-demo-site-companion.php

```diff
@@ -123,3 +125,9 @@ function studio_companion_enqueue_scripts() {
 
 // Enable Jetpack Offline mode to avoid collision with production sites
 add_filter( 'jetpack_offline_mode', '__return_true' );
+
+// Load translations
+function studio_companion_load_textdomain() {
+	load_plugin_textdomain( 'studio-companion-plugin', false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
+}
```

Comment:
Let's make sure the languages folder exists.
Currently we are only downloading the .php file and not the whole folder, so the languages won't be included until we deploy the backend updates.

```suggestion
function studio_companion_load_textdomain() {
    $languages_dir = dirname( plugin_basename( __FILE__ ) ) . '/languages/';
    if ( ! file_exists( $languages_dir ) ) {
        return;
    }

    load_plugin_textdomain( 'studio-companion-plugin', false, $languages_dir );
}
```

Reply:
The guard is not necessary.